Jenkinsfile

```diff
@@ -0,0 +1,127 @@
+#!/usr/bin/env groovy
+
+pipeline {
+    agent {
+        label 'slave-group-k8s'
+    }
+
+    environment {
+        GO111MODULE = 'on'
+        KIND_VERSION = 'v0.11.0'
+        METALLB_VERSION = 'v0.9.6'
+        KUBECONFIG = "$WORKSPACE/kind-kube-config.yaml"
+        TESTING_NAMESPACE = 'namespace-for-testing'
+        WATCH_NAMESPACE = 'namespace-for-testing'
+        PATH="/opt/go/bin:$PATH"
+        RUN_SA_OPERATOR = 'true'
+        MAKE_DATADIR_WRITABLE = 'true'
+    }
+
+    options {
+        timeout(time: 60, unit: 'MINUTES')
+        timestamps()
+        buildDiscarder(logRotator(numToKeepStr: '100', daysToKeepStr: '61'))
+    }
+
+    stages {
+        stage('Build') {
+            steps {
+                sh 'go mod vendor'
+                sh 'make lint'
+            }
+        }
+
+        stage('Unit') {
+            steps {
+                sh 'make unit-test'
+            }
+        }
+
+        stage('E2E') {
+            environment {
```

Comment:
@dmvolod I've ported the check for non-code changes from travis.
jenkins seems to be working well

Reply:
Great, @rigazilla thanks
I think we can merge it now and polish later if needed
Do you know how many agents we have for several parallel PR running?

Reply:
> Do you know how many agents we have for several parallel PR running?

Atm only one node is configured to run operator CI